modules/ivc_champva/spec/services/pdf_stamper_spec.rb

```diff
@@ -21,7 +21,7 @@
         stamp_signature
       end
 
-      let(:test_payload) { 'vha_10_7959c' }
+      let(:test_payload) { 'vha_10_7959f_2' }
```

Comment:
Since this change adds stamps to form 7959c, the "no stamps" test needed to be modified to use the last remaining form without stamps: 7959f_2.

Reply:
I can review when this has been resolved